aries-site/src/examples/components/layouts/BoxExample.js

```diff
@@ -1,26 +1,26 @@
 import React, { useContext } from 'react';
-import { Tile } from 'aries-core';
 import { Box, Heading, ResponsiveContext, Text } from 'grommet';
 import { IconGuidelines } from '../../../components';
 
 export const BoxExample = () => {
   const size = useContext(ResponsiveContext);
 
   return (
-      <Tile pad="medium" background="green" width="medium">
-        <Box
-          pad="small"
-          gap={size === 'small' ? 'large' : 'medium'}
-          align="start"
-        >
-          <IconGuidelines size="xlarge" />
-          <Heading level={2} margin="none" responsive={false}>
-            Guidelines
-          </Heading>
-          <Text size="small">
-            This is the ideology and heartbeat of the HPE Design System.
-          </Text>
-        </Box>
-      </Tile>
+    <Box
```

Comment:
I think in terms of the content we are showing, Card would make sense. However, since this is the `Box` example, it feels like we want to use Box in the code. Maybe we should change this example so that we can use Box?

Reply:
I agree, I missed the context when placing the comment. good catch.

Reply:
Great, I will update the example to demonstrate `Box` as a container rather than a card.